Gemfile

```diff
@@ -14,4 +14,8 @@ end
 # your gem to rubygems.org.
 
 # Load Gemfile with dependencies from manageiq
-eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
+MANAGEIQ_GEMFILE = File.expand_path("spec/manageiq/Gemfile", __dir__)
+unless File.exist?(MANAGEIQ_GEMFILE)
+  raise 'You must run `bin/setup` to install manageiq into spec/manageiq as a rails dummy app for this provider engine gem'
```

Comment:
> Would this make more sense at the top of the Rakefile instead? Then when someone tries running the tests, they will get the error there instead.

That was my first approach. But you will run `bundle exec rake`, so without having `spec/manageiq/Gemfile` it would stop at `bundle` already.

`bin/setup` is PORO. 

Are you suggesting in the Gemfile:
```ruby
eval_gemfile(MANAGEIQ_GEMFILE) if File.exist?(MANAGEIQ_GEMFILE)
```
and issue an error in the `Rakefile`?